validator/accounts/v2/cmd_accounts.go

```diff
@@ -26,9 +26,10 @@ this command outputs a deposit data string which is required to become a validat
 				flags.WalletPasswordFileFlag,
 				flags.AccountPasswordFileFlag,
 				flags.NumAccountsFlag,
+				flags.DeprecatedPasswordsDirFlag,
 				featureconfig.AltonaTestnet,
 				featureconfig.OnyxTestnet,
-				flags.DeprecatedPasswordsDirFlag,
```

Comment:
Is it intended (flag that is not related to PR is being removed)?

Reply:
Not being removed, its just shifted up if you look above.

Reply:
Ok, my bad. Approving.